lib/apiservers/portlayer/swagger.json

```diff
@@ -2408,6 +2686,138 @@
 					}
 				}
 			}
+		},
+		"LoggingJoinConfig": {
+			"type": "object",
+			"required": [
+				"handle"
+			],
+			"properties": {
+				"handle": {
+					"type": "object"
```

Comment:
Should this be a string? I know we wanted to pass around a serialized version of the handle. Plus other swagger endpoints use `handle` as a string.